sdcm/cluster.py

```diff
@@ -1066,6 +1068,41 @@ def config_setup(self, seed_address=None, cluster_name=None, enable_exp=True, en
     def download_scylla_repo(self, scylla_repo, repo_path='/etc/yum.repos.d/scylla.repo'):
         self.remoter.run('sudo curl -o %s -L %s' % (repo_path, scylla_repo))
 
+    def install_scylla(self, scylla_repo):
+        self.remoter.run('sudo systemctl stop scylla-server.service', ignore_status=True)
+        self.remoter.run('sudo yum remove -y "scylla*"')
+        self.remoter.run('sudo yum clean all')
+        result = self.remoter.run('ls /etc/yum.repos.d/epel.repo', ignore_status=True)
+        if result.exit_status == 0:
+            self.remoter.run('sudo yum update -y --skip-broken --disablerepo=epel', retry=3)
+        else:
+            self.remoter.run('sudo yum update -y --skip-broken', retry=3)
+        self.remoter.run('sudo yum install -y rsync tcpdump screen wget net-tools')
+        self.download_scylla_repo(scylla_repo)
+        self.remoter.run('sudo yum install -y {}'.format(self.scylla_pkg()))
+        self.remoter.run('sudo yum install -y {}-gdb'.format(self.scylla_pkg()), ignore_status=True)
+
+    def scylla_setup(self, disks=""):
+        if not disks:
+            # detect local-ssd disks
+            result = self.remoter.run('ls /dev/nvme0n*')
+            if result.stdout:
+                disks = ",".join(re.findall('/dev/nvme0n\w+', result.stdout))
+            else:
+                # detect pd-ssd and pd-standard disks
+                result = self.remoter.run('ls /dev/sd[b-z]')
+                disks = ",".join(re.findall('/dev/sd\w+', result.stdout))
+        assert disks != ""
+        self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic eth0 --disks {}'.format(disks))
+        self.remoter.run('sudo sync')
+        self.log.info('io.conf right after setup')
+        self.remoter.run('sudo cat /etc/scylla.d/io.conf')
+        self.remoter.run('sudo systemctl enable scylla-server.service')
+        self.remoter.run('sudo systemctl enable scylla-jmx.service')
+        self.remoter.run('sudo sync')
+        self.remoter.run('sudo rm -rf /var/lib/scylla/commitlog/*')
+        self.remoter.run('sudo rm -rf /var/lib/scylla/data/*')
```

Comment:
How about moving those two lines to the beginning of install_scylla()?
Try to stop scylla-server service and cleanup the original data.

We want to install scylla in a relatively clean environment: no scylla exists, no old data.